### DISCUSSION TITLE
How does it work in practice?

### DISCUSSION THREAD

**Discussion by 14paavang** [JonathanTreffler/backblaze-personal-wine-container#146] [Q&A] [answered]
If I set it up on truenas scale, and then i use external software to backup my laptop to the NAS with truenas scale running on it, will it back up that data ? Basically does it detect truenas scale vdevs as system drives and then back them up?

**Comment by JonathanTreffler** [comment]
it backups the paths you configure. it does not know about zfs vdevs.

**Reply by 14paavang** [reply]
So I can give it a path that leads to one of the folders in a pool and it will hack it up as a hard drive ? Also does the client update itself within the docker? Does that mean the image gets full and things ?

**Reply by JonathanTreffler** [reply, marked as answer]
The Backblaze Client will get updated by us, not Backblaze and only after you pull a new version of the docker (the Backblaze version is pinned at every docker release, you can override this with ENV variables to always get the newest version (may be unstable)).

**Reply by JonathanTreffler** [reply]
> So I can give it a path that leads to one of the folders in a pool and it will hack it up as a hard drive ?

It is all in the README
